salt/wheel/key.py

```diff
@@ -1,6 +1,29 @@
 # -*- coding: utf-8 -*-
 '''
-Wheel system wrapper for key system
+Wheel system wrapper for key system.
```

Comment:
Also, this sentence could probably be improved. It doesn't really say very much to an en user.